src/main/java/mate/jdbc/dao/DriverDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Driver;
+
+public interface DriverDao {
+    Driver create(Driver driver);
+
+    Optional<Driver> read(Long id);
```

Comment:
better rename to get() and getAll()

Reply:
Okay, I'll change it, but I thought we need to use the same names as in the CRUD principles.